web/styles/compose.css

```diff
@@ -829,6 +825,7 @@ textarea.new_message_textarea,
 
     &.dropdown-widget-button {
         padding: 0 6px;
+        border-radius: 4px;
```

Comment:
added to address sharp left cornors.
![image](https://github.com/zulip/zulip/assets/100200105/1ca1838e-2bc8-4df2-b430-c38a587b2237)

Reply:
@amanagr can you review the CSS changes here since it's a change to a component you maintain?

Reply:
We need to deleted the `border-radius: 0 4px 4px 0;` above.